charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -785,61 +786,64 @@ grafana:
   sidecar:
     datasources:
       enabled: true
-      # -- List of default prometheus compatible datasource configurations.
-      # VM `url` will be added to each of them in templates and `type` will be set to defaultDatasourceType if not defined
-      default:
+      # -- List of prometheus compatible datasource configurations.
```

Comment:
Just notice we're adding our own values to the thrid-party chart `grafana`, which I believe we should avoid.
Since those values are only for vm-stack chart templates, and it breaks if the Grafana chart ever uses any of the same value names. 
It would be better to move below datasources from `grafana.sidecar.datasources`.

Reply:
moved to defaultDatasources